scapy/contrib/vxlan.py

```diff
@@ -1,26 +1,40 @@
+#! /usr/bin/env python
 # RFC 7348 - Virtual eXtensible Local Area Network (VXLAN):
 # A Framework for Overlaying Virtualized Layer 2 Networks over Layer 3 Networks
 # http://tools.ietf.org/html/rfc7348
-
-# scapy.contrib.description = VXLAN
+# http://datatracker.ietf.org/doc/draft-quinn-vxlan-gpe/
```

Comment:
This link does not seem uptodate. Is https://datatracker.ietf.org/doc/draft-ietf-nvo3-vxlan-gpe/ the correct one ?